modules/platforms/dotnet/Apache.Ignite.Core/Impl/IgniteUtils.cs

```diff
@@ -231,5 +232,43 @@ public static int EncodePeekModes(CachePeekMode[] modes, out bool hasPlatformCac
             
             return res & ~platformCache;
         }
+
+        /// <summary>
+        /// Converts integer code representation to CacheMode.
+        /// </summary>
+        /// <param name="code">Integer representation.</param>
+        /// <returns>Cache mode.</returns>
+        /// <exception cref="NotSupportedException">If fails.</exception>
+        public static CacheMode CacheModeFromInt(int code)
```

Comment:
Please remove those methods. We should preserve ordinal values for the `CacheMode` enum instead like this:
```
public enum CacheMode
{
  Replicated = 1,
  Partitioned = 2
}
```

Reply:
In particular, this is important to avoid breaking changes in thin client protocol.

Reply:
Fixed.